vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_card.cc

```diff
@@ -110,6 +117,12 @@ void UpholdCard::OnCreateIfNecessary(
 void UpholdCard::Create(
     ledger::ExternalWalletPtr wallet,
     CreateCardCallback callback) {
+  if (!wallet) {
```

Comment:
If `wallet` was a `nullptr` before this defensive code was added, could it have caused any side effects other than a crash?